task/common/machine/script.go

```diff
@@ -37,6 +37,7 @@ sudo tee /usr/bin/tpi-task-shutdown << 'END'
 if ! test -z "$CI"; then
   cml rerun-workflow
 fi
+sleep 20; while pgrep rclone > /dev/null; do sleep 1; done
```

Comment:
Note: that `sleep 20` is not magical, just two times the `sleep 10` from [line 118](https://github.com/iterative/terraform-provider-iterative/pull/377/files#diff-fd1d8ac8ab64f155f5b5a078ae744a577707c4254732ae35597facadd0acdf86R118); ask Nyquist, Shannon and a few others why. 😄